### ISSUE TITLE
Attachment description is not shown

### ISSUE THREAD

**Issue by GiantCrocodile** [silverbulletmd/silverbullet#862]
Hello,

I'm using v0.7.7 and uploaded a screenshot from my Windows 11 clipboard via `CTRL + V`. This works fine and the image is shown. Sadly I don't see any description of the image after typing it.

Code:
![grafik](https://github.com/silverbulletmd/silverbullet/assets/6271057/5c491981-5b46-4081-ab00-16dd9d4095ba)

When I click out of the block:
![grafik](https://github.com/silverbulletmd/silverbullet/assets/6271057/e018bffc-54d2-4817-b0e5-3cba1012281a)

The screenshot afterwards is shown with no issue but the part in the square bracket isn't shown. Sadly I found no information at https://silverbullet.md/Attachments about the syntax and which of the input fields of this syntactical element means what (I suspect first parameter is the description, second one is the filename).

Let me know if I use it wrong or if this is some issue in silverbullet.md or in the used parser lib.

**Comment by GiantCrocodile** [comment]
Nevermind, I just checked it again and it's not about the title nor the description; it's just the alt-tag. Right now, neither a title nor a description is supported right?

**Comment by zefhemel** [comment]
Yeah, the meaning of the `![alt text](bla.jpg)` is indeed simply used as `alt` attribute right now (that shows on hover). However, this we could potentially change into putting this alt text underneath the image the image. I personally wouldn't have a problem with that.

**Comment by GiantCrocodile** [comment]
Thanks for being open for a change on this @zefhemel. I think both should be possible to be set to different values as it serves different use cases:

- alt text: describes the image (what you see) for a blind person
- caption: provides more or different context.

If both tags have the same value, it would mean that blind persons get the same text twice (duplicate content).

**Comment by zefhemel** [comment]
How would you specify these things separately using the image syntax, though?

**Comment by GiantCrocodile** [comment]
Good question and I wasn't certain as well. This looks like a possible way:

https://www.docstomarkdown.pro/add-image-captions-to-images-in-jekyll-blog-posts-with-markdown/

The syntax isn't ideal in my point of view from an esthetical point of view but it works and I suspect it has a technical reason why they did it like that.
